tendrl/commons/objects/__init__.py

```diff
@@ -162,6 +162,16 @@ def save(self, update=True, ttl=None):
                 NS._int.wreconnect()
                 NS._int.wclient.write(item['key'], item['value'], quorum=True)
 
+    def load_all(self):
+        value = '/'.join(self.value.split('/')[:-1])
+        etcd_resp = NS._int.client.read(value)
```

Comment:
@rishubhjain Yes and that generalization is what is required not just because it avoids having to do certain exception handling steps but because any read from etcd should have that resistance(for the moment, it just does a reconnect(creates new etcd client connection) when the connection fails but whenever we add any extra resilience the new channelized approach lets every access to etcd benefit from that resilience) built..